src/App.tsx

```diff
@@ -21,14 +51,13 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos !== null ? <TodoList todos={todos} /> : <Loader />}
```

Comment:
```suggestion
              {todos.length ? <TodoList todos={todos} /> : <Loader />}
```
You need to check that todos is an array. That's all you need to render the conditional component.

Reply:
I'm using a null comparison because we don't need to show the loader if no search string is found

Reply:
in this case best way split Loader and TodoList condition render.